CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # CHANGELOG
 
 
+## v0.6.4
+* Cursor goes to top of content blocks or top of last block when using keyboard arrows to move to the end of the content blocks.
+* Fix formatting of `jwplayer_synced_at`.
+* issue #232: add custom `blockRenderMap` to avoid creation of unsupported block types when pasting content into blocksmith.
```

Comment:
@bhsiao55 oh dang i forgot we were in triniti. can you remove this issue # from the scottish repo? thanks

Reply:
@efjacobson that issue is in triniti https://github.com/triniti/cms-js/issues/232